foundations/azure/referenceImplementations/core/managementGroupTemplates/policyDefinitions/Compliant-MySQLPolicySetDefinition.json

```diff
@@ -17,47 +17,538 @@
+                        "metadata": {
+                            "version": "1.0.0",
+                            "category": "MySQL"
                         },
-                        "then": {
-                            "effect": "[[parameters('effect')]"
+                        "parameters": {
+                            "effect": {
+                                "type": "string",
+                                "defaultValue": "Deny",
+                                "allowedValues": [
+                                    "AuditIfNotExists",
+                                    "Disabled",
+                                    "Deny"
+                                ],
+                                "metadata": {
+                                    "displayName": "Effect",
+                                    "description": "Enable or disable the execution of the policy"
+                                }
+                            }
+                        },
+                        "policyRule": {
+                            "if": {
+                                "field": "type",
+                                "equals": "Microsoft.DBforMySQL/flexibleServers"
+                            },
+                            "then": {
+                                "effect": "[parameters('effect')]",
+                                "details": {
+                                    "type": "Microsoft.DBforMySQL/flexibleServers/keys",
+                                    "existenceCondition": {
+                                        "allOf": [
+                                            {
+                                                "field": "Microsoft.DBforMySQL/flexibleServers/keys/serverKeyType",
+                                                "equals": "AzureKeyVault"
+                                            },
+                                            {
+                                                "field": "Microsoft.DBforMySQL/flexibleServers/keys/uri",
+                                                "notEquals": ""
+                                            },
+                                            {
+                                                "field": "Microsoft.DBforMySQL/flexibleServers/keys/uri",
+                                                "exists": "true"
+                                            }
+                                        ]
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "name": "Deny-MySql-FlexibleServers-Without-CMK"
+                },
+                {
+                    "properties": {
+                        "properties": {
+                            "displayName": "Deny MySQL without An Azure Active Directory administrator",
```

Comment:
Same as above. Must be DINE, but must provide generic login and sid for assigning this at scale.